The red build ([BuildMulti-PR/39619](https://jenkins.kodi.tv/job/BuildMulti-PR/39619/)) is a Gradle cache lock race on the build node, not this change.

17 of the 19 sub-builds passed, including WIN-32, WIN-64, WIN-ARM64, WIN-UWP-64, OSX-64, OSX-ARM64, IOS-ARM64, TVOS, wasm, webos, and all three linux-docker configurations. The two failures are `android-x86-docker` #15513 and `android-arm64-docker` #16763, both with:

```
* What went wrong:
Gradle could not start your build.
> Cannot create service of type BuildSessionActionExecutor ...
   > Could not create service of type FileAccessTimeJournal ...
      > Timeout waiting to lock journal cache (/home/jenkins/.gradle/caches/journal-1).
        It is currently in use by another process.
          Owner PID: 492170
          Our PID: 495745
          Lock file: /home/jenkins/.gradle/caches/journal-1/journal-1.lock
```

Two concurrent Android jobs on the same node contending for the shared `/home/jenkins/.gradle` cache. `android-arm-docker` #17034 ran clear of the contention and passed, which is the tell: if this PR broke Android, all three Android variants would fail, not two.

The failure is also after compilation, not during it. Both failing builds compiled `GUIFontManager.cpp.o` and archived it into `guilib.a`, reached 85% of the C++ build, and then died in `make apk` with `BUILD FAILED in 1m 4s` from Gradle. Neither log contains a compiler or linker diagnostic naming any file this PR touches.

Could someone retrigger it? I do not have permission to do so myself.

--- a/xbmc/guilib/GUIFontManager.cpp
+++ b/xbmc/guilib/GUIFontManager.cpp
@@ -299,9 +299,11 @@ void GUIFontManager::ReloadTTFFonts(void)
       if (!pFontFile || !pFontFile->Load(strPath, newSize, aspect, 1.0f, fontInfo.border))
       {
         delete pFontFile;
-        // font could not be loaded
+        // This font could not be re-rasterised, but the rest still can. It
+        // keeps the CGUIFontTTF it already had, which m_vecFontFiles still
+        // owns, so it stays renderable at the old size.
         CLog::LogF(LOGERROR, "Couldn't re-load font file: '{}'", strPath);
-        return;
+        continue;
       }
 
       m_vecFontFiles.emplace_back(pFontFile);

